src/react/ErrorPage.jsx

```diff
@@ -43,7 +48,21 @@ class ErrorPage extends Component {
                 />
               }
             />
+            <Button
```

Comment:
Maybe it would be better to add a prop to be able to add buttons? Do you think Go Back should always be there?

Reply:
Removed the button for now.